src/server.js

```diff
@@ -373,7 +378,14 @@ function auth(data) {
 		// If webirc is enabled and we do not know this users IP address,
 		// perform reverse dns lookup
 		if (Helper.config.webirc !== null && !client.config.ip) {
-			reverseDnsLookup(socket, client);
+			reverseDnsLookup(socket, client)
+				.then(()=>{
+					init(socket, client);
+				})
+				.catch(e=>{
```

Comment:
I believe `=>` should be surrounded with whitespaces. Once #1231 is merged, rebase on `master`, which I think should detect this :)